skimage/filters/thresholding.py

```diff
@@ -726,6 +727,154 @@ def threshold_triangle(image, nbins=256):
     return bin_centers[arg_level]
 
 
+def threshold_multiotsu(image, nclass=3, nbins=255):
+    """Generates multiple thresholds for an input image. Based on the
+    Multi-Otsu approach by Liao, Chen and Chung.
+
+    Parameters
+    ----------
+    image : (N, M) ndarray
+        Grayscale input image.
+    nclass : int, optional
+        Number of classes to be thresholded, i.e. the number of resulting
+        regions. Accepts an integer from 2 to 5. Default is 3.
+    nbins : int, optional
+        Number of bins used to calculate the histogram. Default is 255.
+
+    Returns
+    -------
+    idx_thresh : (nclass) array
+        Array containing the threshold values for the desired classes.
+    max_sigma : float
+        Maximum sigma value achieved on the classes.
+
+    Notes
+    -----
+        The current implementation takes some time to calculate the
+        thresholds for five classes. 
```

Comment:
I'm not sure to get this note. Why five classes specifically? Also, "some time" sounds quite vague. Perhaps, a better comment would be to indicate the computation time cost of this function?

Reply:
I was discussing this with @emmanuelle today. This implementation is not that efficient yet; I thought it would be nice to explain that, but I'm not sure the right way to... need some time to figure the computational cost. What about removing the note? I'll come back to this implementation soon.